packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -459,7 +459,7 @@ export default class SentenceTokenizer {
 						"sentence-delimiter" !== nextToken.type &&
 						this.isCharacterASpace( nextToken.src[ 0 ] ) ) {
 						// Don't split on quotation marks unless they're preceded by a full stop.
-						if ( this.isQuotation( token.src ) && previousToken.src !== "." ) {
+						if ( this.isQuotation( token.src ) && previousToken && previousToken.src !== "." ) {
```

Comment:
A simple solution that solves the problem.
Small note: in the rest of the `sentenceTokenizer` we use `isUndefined` instead of an implicit check for a variable to be falsy. Might be worth to unify it. But nothin blocking a merge.

Reply:
I think using the `isUndefined` in this line would be an overkill, since it's going to be something like `if ( this.isQuotation( token.src ) && ! isUndefined( previousToken ) && previousToken.src !== "." ) {`

So personally I'd prefer not to change this line :D